openwisp_radius/tasks.py

```diff
@@ -39,3 +49,38 @@ def delete_unverified_users(older_than_days=1, exclude_methods=''):
 @shared_task
 def convert_called_station_id(unique_id=None):
     management.call_command('convert_called_station_id', unique_id=unique_id)
+
+
+@shared_task
+def send_login_email(accounting_data):
+    User = get_user_model()
+    username = accounting_data.get('username', None)
+    org_uuid = accounting_data.get('organization')
+    user = User.objects.filter(username=username).first()
+    if user:
+        organization = (
+            user.openwisp_users_organization.all().filter(id=org_uuid).first()
+        )
+        if organization:
+            from sesame.utils import get_query_string
```

Comment:
why this import here and not above?

Reply:
The app is not loading if imported above.